pinot-common/src/main/resources/logging-severe-only.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2014-2015 LinkedIn Corp. (pinot-core@linkedin.com)
```

Comment:
It'd be better to update the time range to `2014-2018`. The same in `log4j-fatal-only.properties` as well.

Reply:
Seems we don't check license in config files haha